### ISSUE TITLE
Netmiko fails to install on windows due to textfsm/terminal/fcntl import

### ISSUE THREAD

**Issue by ktbyers** [ktbyers/netmiko#659]

**Comment by ktbyers** [comment]
https://github.com/google/textfsm/issues/20

**Comment by ryanbowen** [comment]
Any progress on this?

**Comment by ktbyers** [comment]
You can test this branch:

https://github.com/ktbyers/netmiko/tree/clitable_fix

There is a fix in there. I haven't released it yet, but let me know if it works.

**Comment by ryanbowen** [comment]
I'll give it a test run tomorrow morning when I get back into work. Thanks for the update.

**Comment by mreams13** [comment]
It works for me (Windows 10).

``` mreams@DESKTOP   play  C\..\Lib  python
Python 3.6.4 (v3.6.4:d48eceb, Dec 19 2017, 06:54:40) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import napalm
>>> driver = napalm.get_network_driver('ios')
>>> device = driver(hostname='192.168.1.1', username='mreams', password='password')
>>> device.open()
>>> device.load_replace_candidate(filename='super_candidate_config.txt')
```

**Comment by ryanbowen** [comment]
@ktbyers It works for me also. Thanks for the update.

**Comment by ktbyers** [comment]
Okay, I will see if I can clean it up slightly and get it released.

**Comment by ktbyers** [comment]
This is fixed in Netmiko release 2.0.1
